Cargo.toml

```diff
@@ -29,10 +29,10 @@ winapi = "0.1.17"
 kernel32-sys = "0.1.0"
 
 [dev-dependencies.glium]
-version = "0.2.1"
+version = "^0.3"
 default-features = false
 features = ["gl_read_buffer", "gl_depth_textures"]
 
 [dev-dependencies.cgmath]
+version = "^0.1"
 git = "https://github.com/bjz/cgmath-rs"
```

Comment:
I presume we can get this from crates.io now? Though just a dev-dependency, cargo publish gets angry about git dependencies (or at least it did in beta 1).